app/workers/education_form/forms/va0994.rb

```diff
@@ -51,18 +51,33 @@ def applicant_ssn
     end
 
     def bank_routing_number
-      return 'N/A' if @applicant.bankAccount.blank?
-      value_or_na(@applicant.bankAccount.routingNumber)
+      if @applicant.bankAccount.present? && @applicant.bankAccount.routingNumber.present?
```

Comment:
I think we can use the `safe operator` to simplify these `if` statements:
```suggestion
      if @applicant.bankAccount&.routingNumber.present?
        @applicant.bankAccount.routingNumber
      else
        value_or_na(@applicant.prefillBankAccount&.bankRoutingNumber)
      end
```